### DISCUSSION TITLE
Vendor issues with v3-betaX + No admin

### DISCUSSION THREAD

**Discussion by vanvanni** [filamentphp/filament#7075] [Help]
Hi!

Currently I run into this problem with a clean L10 project. I tried running `composer require filament/filament:"^3.0"` but with a composer error on hand I moved to `composer require filament/filament:v3.0.0-beta3`. This caused the following vendor issues:
```
Installation failed, reverting ./composer.json and ./composer.lock to their original content.
    - league/uri[6.5.0, ..., 6.7.x-dev] require psr/http-message ^1.0 -> found psr/http-message[1.0, 1.0.1, 1.1] but the package is fixed to 2.0 (lock file version) by a partial update and that version does not match. Make sure you list it as an argument for the update command.
    - league/uri 6.8.0 requires psr/http-message ^1.0.1 -> found psr/http-message[1.0.1, 1.1] but the package is fixed to 2.0 (lock file version) by a partial update and that version does not match. Make sure you list it as an argument for the update command.
    - filament/support v3.0.0-beta3 requires symfony/html-sanitizer ^6.1 -> satisfiable by symfony/html-sanitizer[v6.1.0-BETA1, ..., 6.4.x-dev].
    - filament/filament v3.0.0-beta3 requires filament/support v3.0.0-beta3 -> satisfiable by filament/support[v3.0.0-beta3].
    - symfony/html-sanitizer[v6.1.0-BETA1, ..., 6.4.x-dev] require league/uri ^6.5 -> satisfiable by league/uri[6.5.0, ..., 6.8.0].
    - Root composer.json requires filament/filament v3.0.0-beta3 -> satisfiable by filament/filament[v3.0.0-beta3].

Use the option --with-all-dependencies (-W) to allow upgrades, downgrades and removals for packages currently locked to specific versions.

Installation failed, reverting ./composer.json and ./composer.lock to their original content.
```
To avoid this I tried running ` composer require filament/filament:v3.0.0-beta3 -W` and that fixed the install. After that I ran the filament install command but... No /admin. Also not in my route list. What could be the problem here?

**Development Env:**
- Windows 10
- PHP 8.1.10
- NGINX 1.14
Managed by Laragon

**Comment by mstfkhazaal** [comment]
run this command 
php artisan filament:install --panels

You can read the Documentation
https://beta.filamentphp.com/docs/3.x/panels/installation/

**Reply by vanvanni** [reply]
I might needed to also add the command there. But I did run that command. That is what I tried to state with "After that I ran the filament install command"

**Reply by mstfkhazaal** [reply]
Try
https://github.com/filamentphp/filament/discussions/7060#discussioncomment-6503101

**Reply by vanvanni** [reply]
I will tonight at home. But as mentioned this was a clean L10 installation. No other packages have been installed before.

**Reply by mstfkhazaal** [reply]
I test this in clean l10 installation an i get some error

**Reply by vanvanni** [reply]
Just tested it nothing helps. Even removing vendor folder and upgrading to beta4 with:
```
composer require filament/filament:v3.0.0-beta4
```

**Reply by danharrin** [reply]
This seems to be a Symfony issue that only happens when you have `minimum-stability: beta` - it works with `minimum-stability:dev`

**Reply by vanvanni** [reply]
Quite interesting :D did not expect that. Ill give it a shot later today, thanks!

**Reply by danharrin** [reply]
Yeah I really hope that Symfony has resolved this by the time we release the full version

**Comment by sbc640964** [comment]
I just installed 3.0-beta4 on Laravel v10.15.0 (PHP v8.2.3)
First I can't run `php artisan make:filament-user`
![image](https://github.com/filamentphp/filament/assets/46646044/d1d818d8-326b-4fa6-84a0-5a3be4bd31ad)

Second, the default panel is path `admin` and I get a 404 on `http://localhost:8000/admin`

**Reply by danharrin** [reply]
You need to add the AdminPanelProvider to config/app.php
